src/header/Header.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import menu from "../../src/assets/menu.png";
+
+function Header() {
+  let menuHandler = () => {
```

Comment:
Don't forget we're working with React 🙂 

You can implement the whole open/close functionality much easier. You can manage the open/close state with `useState` and show/hide the whole component with conditional rendering. You can get rid of these event handlers and simply re-write the logic to operate based on React functions. 

Regarding classes – you can take a look at [classnames](https://github.com/JedWatson/classnames) library. You will be able to write something like `className={cx('some-class', { 'some-other-class': true })}`.